packages/aws-cdk/test/api/resource-import/import.test.ts

```diff
@@ -370,10 +381,12 @@ test('do not ask for second part of compound identifier if the user skips the fi
  */
 async function importTemplateFromClean(stack: ReturnType<typeof testStack>) {
   givenCurrentStack(stack.stackName, { Resources: {} });
-  const importer = new ResourceImporter(stack, deployments);
+  const importer = new ResourceImporter(stack, props);
   const { additions } = await importer.discoverImportableResources();
   const importable = await importer.askForResourceIdentifiers(additions);
-  await importer.importResourcesFromMap(importable, {});
+  await importer.importResourcesFromMap(importable, {
+    toolkitStackName: 'CDKToolkit',
```

Comment:
seems like this is required now. similar to the other comment, im iffy on why that's necessary.

Reply:
Hmm. We currently have `toolkitStackName` passed through from the CLI Action options all the way down and at every level it's optional. A few times when the value is needed we add a default fallback. I'm annoyed that this exists so many times.

But it's even worse - it doesn't even look used.